css/app.css

```diff
@@ -207,7 +233,7 @@ section.speakers {
           justify-content: center;
 }
 
-section.speakers h2 {
+h2 {
   font-size: 1.563rem;
   letter-spacing: 0.05em;
   font-family: "Rubik", sans-serif;
```

Comment:
worried ako with global css like this

No need to change but I want you to know that this will affect all h2 so be careful

Reply:
Got this. I targetted yung mismong h2 element since based sa design mukhang same lang style niya sa lahat pero yes, I agree na it would be better if I avoid doing this :) Thank you!